### DISCUSSION TITLE
Delete all feeds in a category?

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6126] [Q&A]
**Is your feature request related to a problem? Please describe.**
As bulk edit of feed title is not easy, I was considering exporting an opml file and then editing the text file and then importing it back to Freshrss. Is it possible to delete all feeds in a category with one click?

**Describe the solution you’d like**
May be you could provide these options by right clicking the category in global view or sidebar of normal view.

**Describe alternatives you’ve considered**

**Additional context**

**Comment by Alkarex** [comment]
Check the *Empty* button in a category settings:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/5bf8e53c-62fc-4dfb-9910-a036b9b490df)
